easybuild/easyblocks/w/wien2k.py

```diff
@@ -157,10 +169,16 @@ def configure_step(self):
             'COMPILERC': os.getenv('CC'),
             'COMPILER': os.getenv('F90'),
             'COMPILERP': os.getenv('MPIF90'),
-        }
+            }
```

Comment:
Hmm, why change this indentation? Does `flake8` complain about this, I don't think it does?

Reply:
it wasn't on purpose (or maybe `emacs` was upset ;)), will change it back